src/mapd-crossfilter.js

```diff
@@ -1038,9 +1046,9 @@ export function replaceRelative(sqlStr) {
         }
 
         if (_orderExpression) { // overrides any other ordering based on dimension
-          query += " ORDER BY " + _orderExpression + ascDescExpr;
+          query += " ORDER BY " + _orderExpression + ascDescExpr + _nullsOrder;
         } else if (dimensionExpression)  {
-          query += " ORDER BY " + dimensionExpression + ascDescExpr;
+          query += " ORDER BY " + dimensionExpression + ascDescExpr + _nullsOrder;
```

Comment:
I would be better to have something less error prone, perhaps two methods `nullsFirst` and `nullsLast` so that the use doesn't have to know that they must feed it the string `" NULLS LAST"` or `" NULLS FIRST"` (including the initial space). Either way, this works for what you're trying to do, and crossfilter isn't build for ease of use, so not so bad